CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.26.74](https://github.com/brave/brave-browser/releases/tag/v1.26.74)
+
+ - Renamed “Brave Search beta” to “Brave” in several locations including onboarding and the URL bar. ([#16564](https://github.com/brave/brave-browser/issues/16564))
```

Comment:
URL bar doesn't show Brave on Android. Probably change to `Renamed "Brave Search beta" to "Brave" in settings and onboarding screen.